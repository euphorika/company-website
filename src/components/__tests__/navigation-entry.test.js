import React from "react"
import renderer from "react-test-renderer"

import NavigationEntry from "../navigation-entry"

describe("current NavigationEntry", () => {

  it("renders span correctly", () => {
    const onClickHandlerDummy = () => {}
    const tree = renderer
      .create(<NavigationEntry
        linkTo="/"
        name="Home"
        onClickHandler={onClickHandlerDummy}
      />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  it("renders link correctly", () => {
    const onClickHandlerDummy = () => {}
    const tree = renderer
      .create(<NavigationEntry
        linkTo="/not-home/"
        name="Not Home"
        onClickHandler={onClickHandlerDummy}
      />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
