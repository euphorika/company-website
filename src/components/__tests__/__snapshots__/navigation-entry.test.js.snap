// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`current NavigationEntry renders link correctly 1`] = `
<a
  href="/not-home/"
  onClick={[Function]}
>
  Not Home
</a>
`;

exports[`current NavigationEntry renders span correctly 1`] = `
<span>
  Home
</span>
`;
